sigle/src/modules/publicHome/components/PublicHome.tsx

```diff
@@ -116,56 +106,61 @@ export const PublicHome = ({ file, settings }: PublicHomeProps) => {
           cardType: 'summary',
         }}
       />
-
-      <Container>
-        <AppHeader />
-        <Header>
-          {settings.siteLogo && (
-            <HeaderLogo src={settings.siteLogo} alt={`${siteName} logo`} />
+      <AppHeader />
+      <Header>
```

Comment:
I think the header should be inside a container, big images won't overflow that way and on mobile you have some padding around :)

Reply:
ah yes that makes sense - will update :)